packaging/rest-service/files/usr/lib/systemd/system/cloudify-api.service

```diff
@@ -19,7 +19,7 @@ ExecStart=/bin/sh -c '/opt/manager/env/bin/gunicorn \
     --timeout 300 cloudify_api.main:app \
     --log-file /var/log/cloudify/rest/api-gunicorn.log \
     --access-logfile /var/log/cloudify/rest/api-audit.log \
-    --access-logformat "%(t)s %(h)s %({X-Cloudify-Audit-Username}o)s %({X-Cloudify-Audit-Tenant}o)s %({X-Cloudify-Audit-Auth-Method}o)s \"%(r)s\" %(s)s %(b)s \"%(a)s\" took %(M)sms"'
+    --access-logformat "%(t)s %(h)s %({X-Cloudify-Audit-Username}o)s %({X-Cloudify-Audit-Tenant}o)s %({X-Cloudify-Audit-Auth-Method}o)s ""%(r)s"" %(s)s %(b)s ""%(a)s"" took %(M)sms"'
```

Comment:
Shouldn't there be a doubled quote after the  ms?

Reply:
no, it's fine

Reply:
Ah, OK- green then.